qa/L0_metrics/metrics_config_test.py

```diff
@@ -97,6 +98,18 @@ def test_cache_counters_missing(self):
         for metric in CACHE_COUNTER_PATTERNS:
             self.assertNotIn(metric, metrics)
 
+    # Histograms
+    def test_inf_histograms_decoupled_exist(self):
+        metrics = self._get_metrics()
+        for metric in INF_HISTOGRAM_DECOUPLED_PATTERNS:
+            for suffix in ["_count", "_sum", ""]:
+                self.assertIn(metric + suffix, metrics)
```

Comment:
Nice work adding checks that the count, sum and bucket metrics exist!

This does not check if the value on each bucket is correct, but you mentioned there is an existing test that will verify the value is correct, so you are not verifying the value here because the existing test will verify the Prometheus histogram metrics is functioning correctly and the tests on histogram_metrics_test.py is verifying the numbers provided to Prometheus is correct via count and sum.

Reply:
Since this test only checks if specific histograms exist without updating the metrics, all values should be 0. For tests regarding Prometheus metrics APIs and functionalities, please refer to 
https://github.com/triton-inference-server/core/blob/main/src/test/metrics_api_test.cc

Reply:
```
# Prometheus histogram buckets are tested in metrics_api_test.cc::HistogramAPIHelper
```

Just a nit for clarification: The Metrics API tests (ex: `TRITONSERVER_MetricObserve`) aren't going through the same code path that the built-in latency metrics are going through. So although there are custom metrics tests testing histograms, they aren't necessarily testing the built-in histogram latency metrics added here.

The built-in metrics actually just use prometheus APIs directly and some C++ helper functions around them today. Ideally, we would unify these to all use the `TRITONSERVER_Metric` layer in the same way for both built-in metrics and custom metrics for easier test coverage and maintenance in the future.

---

All that being said, I think the current tests included here with the checks around the `_sum` value are fine for now for this PR. When adding other histogram latency metrics, I think it would be a good idea to either (a) add some bucket-related tests to these python unit tests or (b) attempt to unify the internal metrics to use the same custom metrics APIs or internal C++ classes around them for better re-use of the `metrics_api_test.cc` tests.